### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by fbertsch**
Show all labels for categorical histograms, and improve errors for telemetry.js

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/telemetry-dashboard/279)

### CODE REVIEW

**Comment by fbertsch** [comment]
@chutten r?

File: new-pipeline/src/dist.js [hunk 1]
```diff
@@ -1163,12 +1168,12 @@ function displaySingleHistogramSet(axes, useTable, histograms, title,
       max_y: maxPercentage + 2, // Add some extra space to account for the top label
       transition_on_update: false,
       target: axes,
-      x_label: histogram.description,
```
Comment (chutten) [inline new-pipeline/src/dist.js:1166]: Why did you remove the description from the x_label?

File: v2/telemetry.js [hunk 2]
```diff
@@ -23,17 +23,21 @@
   Telemetry.Histogram = (function () {
     function Histogram(buckets, values, kind, submissions, sum,
       description, measure) {
-      assert(["number", "string"].indexOf(typeof buckets[0]) > -1, "`buckets` must be an array, is " + (typeof buckets[0]));
-      assert(typeof values[0] === "number", "`values` must be an array");
+      assert(["number", "string"].indexOf(typeof buckets[0]) > -1,
+          "`buckets` must be an array of numbers or strings, is " + (typeof buckets[0]));
```
Comment (chutten) [inline v2/telemetry.js]: Probably should put "array of" at the end there, or wrap it in []

File: v2/telemetry.js [hunk 3]
```diff
@@ -23,17 +23,21 @@
...
+      assert(typeof values[0] === "number",
+         "`values` must be an array of numbers, is " + (typeof values[0]));
```
Comment (chutten) [inline v2/telemetry.js]: ditto. "is array of"

**Review by chutten — COMMENTED** [review]
A few nits, though I'm wondering why you ditched the description.

File: new-pipeline/src/dist.js (hunk 1 above)
Comment (fbertsch) [reply new-pipeline/src/dist.js:1166]: The description was overlapping the labels. Plus we already show the description above the histogram, the one below was superfluous.

Comment (chutten) [reply new-pipeline/src/dist.js:1166]: You could do what I did with the grouped barchart and put a bottom of 150 on ones that need it.

Comment (chutten) [reply new-pipeline/src/dist.js:1166]: I dislike unlabeled axes. Can you just drop descriptions from the categoricals?

**Comment by fbertsch** [comment]
Okay, now it shows all labels, with the description, and a greater space on bottom for categorical histograms. @chutten RFAL
